bigquery_etl/cli/backfill.py

```diff
@@ -177,50 +176,24 @@ def validate(
     project_id,
 ):
     """Validate backfill.yaml files."""
-    backfill_files = []
-
-    if qualified_table_name:
-        try:
-            match = QUALIFIED_TABLE_NAME_RE.match(qualified_table_name)
-            project_id = match.group("project_id")
-            dataset_id = match.group("dataset_id")
-            table_id = match.group("table_id")
-        except AttributeError:
-            click.echo(
-                "Qualified table name must be named like:"
-                + " <project>.<dataset>.<table>"
-            )
-            sys.exit(1)
-
-        path = Path(sql_dir)
-        query_path = path / project_id / dataset_id / table_id
-
-        if not query_path.exists():
-            click.echo(f"{project_id}.{dataset_id}.{table_id}" + " does not exist")
-            sys.exit(1)
-
-        backfill_file = path / project_id / dataset_id / table_id / BACKFILL_FILE
-        backfill_files.append(backfill_file)
+    backfills_dict = get_backfill_file_to_entries_map(
+        sql_dir, project_id, qualified_table_name
+    )
 
-    else:
-        backfill_files = paths_matching_name_pattern(
-            None, sql_dir, project_id, [BACKFILL_FILE]
-        )
+    backfill_files = list(backfills_dict.keys())
 
-    for file in backfill_files:
+    for backfill_file in backfill_files:
```

Comment:
Didn't see any list operations besides iteration in this block, maybe just a `for backfill_file in backfills_dict: ...` would work here?